spec/lib/schema_consistency_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'spec_helper'
+require 'govuk_schemas'
+require "finder_schema_converter"
+
+RSpec.describe "the finder schemas" do
+  def specialist_document_schema_definitions
+    GovukSchemas::Schema.find(publisher_schema: "specialist_document")["definitions"]
+  end
+
+  context "all facets on finder schema should appear as metadata in the specialist document schema" do
+    FinderSchemaConverter.except_ignored_finders.each do |finder_schema|
```

Comment:
The selection, looping and reading of the finders to test could be moved into here to make `FinderSchemaConverter` do less. You could instantiate the `FinderSchema` in here.